### PR TITLE
Update auth screen to be own component and render conditional message

### PR DESCRIPTION
**Pull request by jbt**
*Issue #, if available:*

*Description of changes:*

* Moves authenticator into own component now using SecndaryLayout
* Minor styling tweaks to remove box shadow and use consistent link color, but not introduce any further CSS hacking inside authenticator
* Watches the display state of the authenticator form and conditionally renders notice above the sign up form

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/web-ui/src/public/Auth.vue [hunk 1]
```diff
@@ -0,0 +1,166 @@
+<template>
+  <secondary-layout>
```
Comment (kazbaig) [inline src/web-ui/src/public/Auth.vue:2]: Nit pick: can we use `SecondaryLayout` for casing? I know it's not the `vue` way, but it's the naming convention we've followed thus far. Perhaps something to consider changing throughout the app.

File: src/web-ui/src/public/Auth.vue [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+              type: 'string',
+              required: true
+            },
+          ]
+        }
+      }
+    }
+  },
+  mounted() {
+    this.$refs['authenticator'].$watch('displayMap', (newVal) => {
```
Comment (kazbaig) [inline src/web-ui/src/public/Auth.vue:57]: Another nit pick - dot notation would be ideal here. As it stands, might mess with IDE's renaming feature down the line if needed.

File: src/web-ui/src/public/Welcome.vue [hunk 3]
```diff
@@ -32,7 +32,7 @@
           <div class="login-cta d-flex justify-content-center align-items-center">
             Have an account?<router-link to="/auth" class="sign-in btn btn-link">Sign in</router-link>
           </div>
-          <router-link to="/auth" class="create-account mt-3 mt-sm-0 ml-sm-3 btn btn-primary"
+          <router-link to="/auth?signup=true" class="create-account mt-3 mt-sm-0 ml-sm-3 btn btn-primary"
```
Comment (kazbaig) [inline src/web-ui/src/public/Welcome.vue:35]: Since we've made the change the comment on line 29 is talking about, let's remove that comment as well.

**Comment by kazbaig** [comment]
LGTM!

**Review by james-jory — APPROVED** [review]

**Pull request merged**